src/app/core/auth/auth.service.ts

```diff
@@ -337,7 +339,7 @@ export class AuthService {
   /**
    * Redirect to the route navigated before the login
    */
-  public redirectToPreviousUrl() {
+  public redirectToPreviousUrl(isStandalonePage: boolean) {
```

Comment:
Would it be a good idea to rename this method to something like `redirectAfterLoginSuccess` -- especially since the redirect is actually not to previous url in all cases?

Reply:
I agree

Reply:
@mspalti please consider that in the auth service SSR instance there the same [redirectToPreviousUrl](https://github.com/mspalti/dspace-angular/blob/6425f6e7d1542c08c3ab0d9d187e1158d29eb0e4/src/app/core/auth/server-auth.service.ts#L57) that needs to be fixed too

Reply:
Updated the method name.